core/src/main/scala/chisel3/experimental/verification/package.scala

```diff
@@ -8,36 +8,52 @@ import chisel3.internal.firrtl.{Formal, Verification}
 import chisel3.internal.sourceinfo.SourceInfo
 
 package object verification {
+
+  /** Named class for assertions. */
+  final class Assert(predicate: Bool, msg: String = "") extends BaseSim
+
+  /** Named class for assumes. */
+  final class Assume(predicate: Bool, msg: String = "") extends BaseSim
+
+  /** Named class for covers. */
+  final class Cover(predicate: Bool, msg: String = "") extends BaseSim
+
```

Comment:
Do you want the `predicate` and `msg` to be accessible? If so they should be `vals` (`case classes` make their parameters vals automatically, but since we changed them to regular classes, this is no longer true).

Reply:
Please don't make `msg` accessible. That will just make it harder to deprecate it later.

Reply:
i will just remove `msg`, since we're not doing anything with these classes yet.